AUTHORING_GUIDE.md

```diff
@@ -481,7 +481,24 @@ __Note:__ As a temporary workaround, each project currently uses first
 `noxfile-template.py` found in a parent folder above the current sample. In
 order to simulate this locally, you need to copy + rename the parent
 `noxfile-template.py` as `noxfile.py` in the folder of the project you want to
-run tests.
+run tests:
+
+```console
+cd python-docs-samples
+cp noxfile-template.py PRODUCT/noxfile.py
+cd PRODUCT/
+nox -s lint
+```
+or if you use subdirectories that contain `requirements.txt`, you'll need to
+copy it into each subdirectory and run lint from there:
+
+```console
+cd python-docs-samples
+cp noxfile-template.py PRODUCT/EXAMPLE/noxfile.py
+cd PRODUCT/EXAMPLE/
+nox -s lint
```

Comment:
Can we condense these for brevity? I think the confusion is around what's considered a "project" in python, so we can clarify that above.

```suggestion
```console
cd python-docs-samples
cp noxfile-template.py PATH/TO/YOUR/PROJECT/noxfile.py
cd PATH/TO/YOUR/PROJECT/
```